lib/node/query.js

```diff
@@ -180,9 +180,6 @@ var Query = Node.define({
     var self = this;
 
     var args = sliced(arguments);
-    if (o.length == 0) {
```

Comment:
Removing fix placed in https://github.com/brianc/node-sql/pull/197 which inserts default values.